src/prefect/orion/schemas/actions.py

```diff
@@ -314,6 +314,13 @@ class BlockDocumentUpdate(ActionBaseModel):
     data: dict = FieldFrom(schemas.core.BlockDocument)
 
 
+@copy_model_fields
+class BlockDocumentReplace(ActionBaseModel):
```

Comment:
Similar to my comment above, our standard here is `Update` for all these actions, we should probably stick to that. We can document that it will not merge with existing data.

Reply:
I think Alex wanted a separate endpoint for `post` so I copied `update_block_document` (which has `patch`) with `replace_block_document`. Then for consistency, I also added this model, but unsure if it's necessary.